src/main/kotlin/org/jitsi/nlj/transform/node/outgoing/TccSeqNumTagger.kt

```diff
@@ -47,7 +50,8 @@ class TccSeqNumTagger(
                         ?: rtpPacket.addHeaderExtension(tccExtId, TccHeaderExtension.DATA_SIZE_BYTES)
 
                     TccHeaderExtension.setSequenceNumber(ext, currTccSeqNum)
-                    transportCcEngine?.mediaPacketSent(currTccSeqNum, rtpPacket.length.bytes)
+                    val len = rtpPacket.length.bytes
+                    packetInfo.onSent { weakTcc.get()?.mediaPacketSent(currTccSeqNum, len) }
```

Comment:
Is `currTccSeqNum` going to be a problem here?  It'll capture the reference, right?

Reply:
I confess that capture-by-value or capture-by-reference always confuses me -- but yes, I think you're right.